src/cets.erl

```diff
@@ -527,3 +552,13 @@ call_user_handle_down(RemotePid, _State = #{tab := Tab, opts := Opts}) ->
         _ ->
             ok
     end.
+
+-type start_error() :: bag_with_conflict_handler.
+-spec check_opts(start_opts()) -> [start_error()].
+check_opts(Opts) ->
+    check_bag_with_conflict_handler(Opts).
```

Comment:
Minor, style: this extra function is not needed, I would put the logic from lines 561-564 directly here. The clause in line 561 is just as informative as the function name.

Reply:
could be.
I just was thinking to add more check_* parameters later, if needed so it could be:

```
check_opts(Opts) ->
    check_bag_with_conflict_handler(Opts)
    ++ check_some_other_option(Opts)
    ...
```

Reply:
ok, replaced